### DISCUSSION TITLE
How to rename generated "index.html" file

### DISCUSSION THREAD

**Discussion by arcadeJHS** [withastro/docs#2155] [Ideas]
Is it possible, after "npm run build", generate a "default.aspx" file in place of "index.html" inside the "dist" folder?
Thank you!

**Comment by delucis** [comment]
Hi @arcadeJHS! Just a note to say this is the discussions area for Astro’s documentation website — you’ll probably get more eyes and more chance of a helpful answer in the `#support` channel on Discord. We’d be happy to have you there: https://astro.build/chat

**Comment by arcadeJHS** [comment]
I see! Thank you and sorry to post in the wrong place!

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
